readme.md

```diff
@@ -0,0 +1,63 @@
+### WordCamp.org
+
+This repository mirrors https://meta.svn.wordpress.org/sites/trunk/wordcamp.org/ and can be used to contribute used Git instead of SVN.
+
+###### Setup 
+
+There are two primary ways to setup this repo for local development.
+
+1. Using VVV. WordCamp.org is available as part of WordPress.org's meta network, and can be setup by following meta-environment setup guide: https://github.com/WordPress/meta-environment/blob/master/docs/install.md
+
+1. If you'd like to only setup WordCamp.org and would like to use Docker, you can use inbuilt docker provisioning. Please follow these steps
+
+    **Note:** This will create `.data` directory which will contain MySQL files to persist data across docker restarts.
+
+    1. Make sure you have Docker installed and Docker daemon running on your system.
+    
+    1. Clone the repo: 
+        ```
+        git clone git@github.com:WordPress/wordcamp.org.git
+        ```
+    
+    1. Change into the directory, and run docker compose command. This could take some time depending upon the internet speed.
+        ```
+        cd wordcamp.org && docker-compose up --build
+        ```
+         This will also start the docker environment for WordCamp.org development.
+         
+    1. Add the following resolutions to your host file:
+        ```
+        127.0.0.1	central.wordcamp.test
+        127.0.0.1	wordcamp.test
+        127.0.0.1	2014.content.wordcamp.test
+        127.0.0.1	2014.misc.wordcamp.test
+        127.0.0.1	2014.new-site.wordcamp.test
+        ```
+        
+        `/wp-admin` pages for these sites should now be accessible. Use `admin` as username and `password` as password to login.
+        
+        **Note:** `https` URL scheme must be used to visit these sites. Security exception will be required in first time run.
```

Comment:
`MIGHT` - It'd be nice to add instructions for adding the cert to the browser, so you get a green link instead of yellow. That's not necessary, just a nice-to-have

Reply:
Punting this for later. Because the process will actually be other way around. You would generate a cert on host first and then add it inside Docker. 
Keeping this conversation un-resolved for now, and will open a issue when we merge this.

Reply:
👍

Reply:
Raised https://github.com/WordPress/wordcamp.org/issues/121